src/explore-education-statistics-admin/src/pages/admin-dashboard/components/MethodologySummary.tsx

```diff
@@ -96,118 +96,127 @@ const MethodologySummary = ({
-                {methodology.permissions.canDeleteMethodology && (
-                  <Button
-                    onClick={() =>
-                      setDeleteMethodologyDetails({
-                        methodologyId: methodology.id,
-                        amendment: methodology.amendment,
-                      })
-                    }
-                    className="govuk-button--warning"
-                  >
-                    {methodology.amendment ? 'Cancel amendment' : 'Remove'}
-                  </Button>
-                )}
+                          {canEdit
+                            ? 'Edit this amendment'
+                            : 'View this amendment'}
+                        </ButtonLink>
+                        <ButtonLink
+                          to={generatePath(methodologySummaryRoute.path, {
+                            publicationId,
+                            methodologyId: methodology.previousVersionId,
+                          })}
+                          className="govuk-button--secondary govuk-!-margin-left-4"
+                        >
+                          View original methodology
+                        </ButtonLink>
+                      </ButtonGroup>
+                    </>
+                  ) : (
+                    <>
+                      <ButtonGroup>
+                        <ButtonLink
+                          to={generatePath(methodologySummaryRoute.path, {
+                            publicationId,
+                            methodologyId: methodology.id,
+                          })}
+                        >
+                          {canEdit
+                            ? 'Edit this methodology'
+                            : 'View this methodology'}
+                        </ButtonLink>
+                        {methodology.permissions
+                          .canMakeAmendmentOfMethodology && (
+                          <Button
+                            type="button"
+                            onClick={() =>
+                              setAmendMethodologyId(methodology.id)
+                            }
+                            variant="secondary"
+                          >
+                            Amend methodology
+                          </Button>
+                        )}
+                      </ButtonGroup>
+                    </>
+                  )}
+                </div>
+                <div className="govuk-grid-column-one-third dfe-align--right">
+                  {methodology.permissions.canDeleteMethodology && (
+                    <Button
+                      onClick={() =>
+                        setDeleteMethodologyDetails({
+                          methodologyId: methodology.id,
+                          amendment: methodology.amendment,
+                        })
+                      }
+                      className="govuk-button--warning"
```

Comment:
Could use the `variant="warning"` prop instead of this class

Reply:
Thanks, great tip.